azure/lb.sh

```diff
@@ -120,6 +120,25 @@ fn azure_lb_addresspool_delete(name, group, lbname) {
 	)
 }
 
+fn azure_lb_addresspool_get_id(addrpoolname, resgroup, lbname) {
+	out, err <= (
+		az network lb address-pool show
+					--resource-group $resgroup
+					--lb-name $lbname
+					--name $addrpoolname
+					--output json
+					>[2=]
+	)
+
+	if $err != "0" {
```

Comment:
You right. 
I agree that can be weird/nasty only return empty result. What do you suggest for print? Something like "Address-poll with name <addressp-pool-name> not found"

I'll also change to return two values: the address-poll-name and original error code. So, we can remove that empty return, according you suggested.